### DISCUSSION TITLE
Welcome to ConfigLMM Discussions!

### DISCUSSION THREAD

**Discussion by davispuh** [ConfigLMM/ConfigLMM#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by spcano01** [comment]
Saw your post on reddit, going to give this a try.  Nicely done going over 100 stars, but wondering is this still being developed and if so, any roadmap?  

Finally got ansible and terraform working, but like you mentioned, a bit of a pita.  Even considering chezmoi/mise/fnox to replace ansible.  

So many options, good to have more especially if new ones are much simpler, or for homepage vs enterprise.

**Reply by davispuh** [reply]
> going to give this a try

Use latest git master, don't use last release/published RubyGem. There's been way too many changes and improvements since then. And let me know about any issues you encounter.

> wondering is this still being developed and if so, any roadmap?

Yeah I'm actively developing it.
I just hadn't pushed changes to GitHub but I just did that with 84 commits:
`106 files changed, 2529 insertions(+), 968 deletions(-)`

 

Roadmap is based on what I need. It's a massive project so I'm implementing what I will be using myself. Would be really great to get other contributors :yum: 

What kind of things do you want to install/deploy/configure? You can list all software, distros etc everything you dream of :)


> if new ones are much simpler, or for homepage vs enterprise.

Basically what I'm trying to do with ConfigLMM is hide complexity inside tool itself rather than in configuration files how it's with other tools. And I'm not limiting ConfigLMM usage just for self-hosting but it should be usable for large enterprises aswell. In fact I'm actually using it for my own company to configure all infrastructure (all servers/VMs).
